src/components/Text/Text.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { TextProps } from "./types";
+import "./styles.scss";
```

Comment:
since we have another styles.scss file, what do you think about naming the associated stylesheet after the Component it is for?

Reply:
Down

Reply:
Actually, i dont think it matters much. Was hoping we could maybe set a precedent where each component folder follows the same structure:

ComponentName [folder]
- ComponentName.tsx
- styles.scss
- types.ts
- utils.ts


What do you think?

Reply:
I just think when it comes to working on stuff having X number of styles and types folders will be inconvenient - naming the associated Types and Styles files after their component name would only require us to know the associated component and the extension of the file we're looking for which I think will be easier to maintain going fwd. Not that this site will even get too large but in terms of establishing a precedent I think it could be helpful

Reply:
I also think utils should live in the project root - I find when working on sites that oftentimes utility functions are used in several places across the app, so this would avoid us both A) needing to copy functions to different utils files, and B) needing to grab another component's utils file when needing the same function

Reply:
Yeah i definitely agree on the utils point, for global utils it makes sense for those to live in a global util file rather than copy pasting to multiple files. Although to avoid super long files as the project grows, I think some degree of isolation can be helpful